src/noInputPrefixRule.ts

```diff
@@ -0,0 +1,46 @@
+import * as Lint from 'tslint';
+import * as ts from 'typescript';
+import { sprintf } from 'sprintf-js';
+import { NgWalker } from './angular/ngWalker';
+
+export class Rule extends Lint.Rules.AbstractRule {
+  public static metadata: Lint.IRuleMetadata = {
+    ruleName: 'no-input-prefix',
+    type: 'maintainability',
+    description: 'Name inputs without the prefix with the provided prefixes',
```

Comment:
can you propose something else ?

Reply:
Inputs names should not be prefixed with the configured disallowed prefixes.

What do you thunk about this?